chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -304,7 +303,7 @@ curl -s -o /mnt/usr/sbin/crowbar_join <%= @crowbar_join %>
 chmod +x /mnt/usr/sbin/crowbar_join
 
 mkdir -p /mnt/var/log/crowbar
-post_state $HOSTNAME "installed"
+crowbarctl node transition $HOSTNAME "installed"
```

Comment:
OK, this does not work this way. It's not possible to simple call `/mnt/usr/bin/crowbarctl` due to missing dependencies.

It might be possible if the post-script itself is chrooted  (`<chrooted config:type="boolean">false</chrooted>`) and I do not know why don't we have it...

Reply:
Or I might add extra chrooted script that will only call crowbarctl from the new installed system...

Reply:
Have you tested this and is it working? Your comment seems to suggest that it's not quite there yet :smile:

Reply:
Oh - the comment is obsolete, I've addressed this in 2nd commit.

Reply:
Oh, sorry. That's sometimes a little hard to tell on GitHub.